src/ui/suspense/nodeCaches.ts

```diff
@@ -173,27 +160,157 @@ export const nodeDataCache: Cache<
   },
 });
 
+export const processedNodeDataCache: Cache<
+  [replayClient: ReplayClientInterface, pauseId: PauseId, nodeId: string],
+  NodeInfo | null
+> = createCache({
+  config: { immutable: true },
+  debugLabel: "ProcessedNodeData",
+  getKey: ([replayClient, pauseId, nodeId]) => `${pauseId}:${nodeId}`,
+  load: async ([replayClient, pauseId, nodeId]): Promise<NodeInfo | null> => {
+    if (!pauseId || typeof nodeId !== "string") {
```

Comment:
This is unexpected, given the way the types are defined for this cache:
```js
  [replayClient: ReplayClientInterface, pauseId: PauseId, nodeId: string],
```
Which is wrong?